app/src/main/java/agency/five/codebase/android/movieapp/mock/FavoritesDBMock.kt

```diff
@@ -0,0 +1,20 @@
+package agency.five.codebase.android.movieapp.mock
+
+import kotlinx.coroutines.flow.*
+
+object FavoritesDBMock {
+    private val _favoriteIds = MutableStateFlow(MoviesMock.getMoviesList().filter { movie -> movie.isFavorite }
+        .map { movie -> movie.id }.toSet())
+    val favoriteIds: StateFlow<Set<Int>> = _favoriteIds
+
+    fun insert(movieId: Int) {
+        if (favoriteIds.value.contains(movieId)) {
+            return
+        }
```

Comment:
You don't need to return here, nothing will happen if you add existing id to set of ids